docs/proposals/scheduling/20220629-schedule-device-in-card-level.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Fine-grained Device Management
+authors:
+- "@buptcozy"
+reviewers:
+- "@hormes"
+- "@allwmh"
+- "@honpey"
+- "@saintube"
+- "@stormgbs"
+- "@zwzhang0107"
+- "@eahydra"
+creation-date: 2022-06-29
+last-updated: 2022-06-29
+status: provisional
+
+---
+
+# Fine-grained Device Management
+
+## Table of Contents
+
+<!--ts-->
+
+* [Fine-grained Device Management](#fine-grained-device-management)
+  * [Summary](#summary)
+  * [Motivation](#motivation-and-user-stories)
+    * [Why Need This Proposal](#why-need-this-proposal)
+      * [NVLink](#nvlink)
+      * [MIG](#mig)
+      * [Oversold](#oversold)
+    * [Goals](#goals)
+  * [Proposal](#proposal)   
+    * [API](#api) 
+    * [Implementation Details](#implementation-details)
+      * [Scheduler](#scheduler)
+      * [Device reporter](#device-reporter)
+      * [RuntimeProxy](#runtimeProxy)
+    * [Compatibility](#compatibility)
+  * [Unsolved Problems](#unsolved-problems)
+  * [Alternatives](#alternatives)
+  * [Implementation History](#implementation-history)
+  * [References](#references)
+  
+
+<!--te-->
+
+## Summary
+support schedule devices like gpu\rdma\fpga in card level for better schedule placement. 
```

Comment:
the summary of this proposal is not about better schedule placement, it's about allowing scheduler to specify which card should be allocated.